src/shared/icon/index.ts

```diff
@@ -1,27 +1,28 @@
 import { html, LitElement } from "lit";
 import { customElement, property } from "lit/decorators.js";
 
-import 'material-symbols/outlined.css';
```

Comment:
this could break the styling of this component when used in isolation. It was necessary to make the tests, build and the devserver work.